internal/cli/cmd/approve/approve.go

```diff
@@ -29,9 +29,17 @@ func NewCommand(cfg config.CLIConfig, opt *option.Option) *cobra.Command {
 	}
 
 	cmd := &cobra.Command{
-		Use:     "approve --project=project --freight=freight --stage=stage",
-		Short:   "Manually approve freight for promotion to a stage",
-		Example: "kargo approve --project=project --freight=abc1234 --stage=qa",
+		Use:   "approve [--project=project] --freight=freight --stage=stage",
+		Short: "Manually approve freight for promotion to a stage",
+		Args:  option.NoArgs,
+		Example: `
+# Approve a freight in a project to the QA stage
```

Comment:
Tricky grammar thing that I find myself stumbling over frequently.

"Freight" is a [mass noun](https://www.grammar-monster.com/glossary/mass_noun.htm) and inherently represents an unspecified volume of -- well, freight.

Grammatically, "a freight" is incorrect. So is "freights." This is pretty esoteric, especially for ESL speakers, so I try not to be dogmatic about this. If it's grammatically incorrect in code or a comment, I don't think it matters. For user-facing things like docs and CLI help, I think we need to exercise more than the usual amount of care.

The tricks I have found help in those scenarios are:

1. Drop the indefinite article "a"
2. Preface "freight" with "a piece of"

So, personally, I would rephrase this either of the following:

* Approve freight in a project for the QA stage
* Approve a piece of freight in a project for the QA stage

Separately, with nearly everything in Kargo being project-scoped, I wonder if we may wish to avoid using the phrase "in a project" unless we truly need to clarify. (e.g. Credentials can exist in a project or globally, but that's an exception.)

So in the end, maybe something like this?

* Approve freight for the QA stage
* Approve a piece of freight for the QA stage

This same comment probably applies in other places.

Also, please, please forgive me if this seems overly pedantic. That's not my intention at all. I think with the task at hand being very much about improving consistency throughout the CLI, these are the exact sort of things we need to hammer out and then whatever we decide, stick with it.

Reply:
> Also, please, please forgive me if this seems overly pedantic.

I actually really like you thinking this through, so I appreciate the suggestions.

---

Within the context of promotion, I think:

> Approve a piece of freight in a project for the QA stage

would fit best because it clarifies it being a singular unit.

I also agree with your suggestion around dropping "a project".